coral/settings.py

```diff
@@ -190,9 +296,31 @@
 # when hosting Arches under a sub path set this value to the sub path eg : "/{sub_path}/"
 FORCE_SCRIPT_NAME = None
 
+FORCE_USER_SIGNUP_EMAIL_AUTHENTICATION = False
```

Comment:
add comment:

this allows us to onboard a user via magic link without allowing self-onboarding